eventmesh-dashboard-common/src/main/java/org/apache/eventmesh/dashboard/common/model/remoting/offset/GetOffsetRequest.java

```diff
@@ -18,12 +18,12 @@
 package org.apache.eventmesh.dashboard.common.model.remoting.offset;
 
 
-import org.apache.eventmesh.dashboard.common.model.metadata.GroupMetadata;
-
 import lombok.Data;
 
 @Data
 public class GetOffsetRequest {
 
-    private GroupMetadata groupMetadata;
+    private String bootStrapServers;
+
+    private String groupName;
```

Comment:
It should be `bootstrap` instead of `bootStrap` as `bootstrap` is a single word.

Is there any relationship between `GetOffsetRequest` and these two members?

Reply:
I'm not sure, what fields are needed for a request, developers who implement these remoting services can modify the fields

Reply:
okay, then lets leave a TODO comment and delete these members.

Reply:
You may finish this part in the next PR.